src/app.ts

```diff
@@ -36,5 +36,6 @@ app.listen(3000, () => {
 });
 
 app.get('/openJobRoles', getAllJobRoles);
+app.get('/openJobRoles/:id', getSingleJobRole);
```

Comment:
getJobRoleById instead?

Reply:
changed the methods around so the one being called here is called getJobRoleById :)